src/rpc/mining.cpp

```diff
@@ -417,6 +417,8 @@ UniValue getblocktemplate(const UniValue& params, bool fHelp)
             "It returns data needed to construct a block to work on.\n"
             "See https://en.bitcoin.it/wiki/BIP_0022 for full specification.\n"
 
+            "\nTo obtain information about founder's reward or funding stream amounts, use 'getblocksubsidy $(getblocktemplate)[\"height\"]'.\n"
```

Comment:
This isn't actually valid shell, just something I threw together in Slack 😝 Let's try this:
```suggestion
            "\nTo obtain information about founder's reward or funding stream\n"
            "amounts, use 'getblocksubsidy HEIGHT' passing in the height returned\n"
            "by this API.\n"
```